pkg/portutil/portutil.go

```diff
@@ -18,14 +18,31 @@ package portutil
 
 import (
 	"net"
-	"strconv"
 	"strings"
 
 	gocni "github.com/containerd/go-cni"
+	"github.com/docker/go-connections/nat"
 	"github.com/pkg/errors"
 )
 
-func ParseFlagP(s string) (*gocni.PortMapping, error) {
+func splitParts(rawport string) (string, string, string) {
```

Comment:
needs comment lines to explain the ret values

Reply:
done